responsibleai/tests/rai_insights/test_rai_insights_validations.py

```diff
@@ -392,6 +394,19 @@ def test_classes_exceptions(self):
         assert 'The train labels and distinct values in target ' + \
             '(train data) do not match' in str(ucve.value)
 
+        with pytest.raises(UserConfigValidationException) as ucve:
+            RAIInsights(
+                model=model,
+                train=X_train,
+                test=X_test,
+                target_column=TARGET,
+                task_type='classification',
+                classes=[0, 1],
+                feature_metadata=FeatureMetadata(
+                    dropped_features=X_train_feature_names))
+        assert 'The train labels and distinct values in target ' + \
+            '(train data) do not match' in str(ucve.value)
```

Comment:
Is the above validation needed or is it just copy/paste error? There seems to be the same assert in line 394-395.

Reply:
that was to test when all features are dropped while feature class value is not matched. This can be removed. Removing